src/System.Net.Primitives/src/ApiCompatBaseline.uap101.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Net.Primitives:
+CannotRemoveBaseTypeOrInterface : Type 'System.Net.Sockets.SocketException' does not inherit from base type 'System.Runtime.InteropServices.ExternalException' in the implementation but it does in the contract.
```

Comment:
> Shouldn't it be complaining about Win32Exception?

Win32Exception exists, but in the implementation it's looking at, Win32Exception does not derive from ExternalException, hence the warning.